src/components/Configure/layout/ConditionalProxyLayout/hasDisqualifyingKey.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * given a list of keys and disqualifying keys,
+ * return true if any of the disqualifying keys are found in the list of keys
+ * @param keys
+ * @param disqualifyingKeys
+ * @returns
+ */
+export function hasDisqualifyingKey(keys: string[], disqualifyingKeys: string[]): boolean {
+  return keys.reduce((hasDisqualifying: boolean, key: string) => {
```

Comment:
nit : I personally find it confusing to use `reduce` when there isn't a need to accumulate an intermediate result. This function would be more readable if implemented with `forEach` but that's just my opinion.

Reply:
Or even simpler would be a one-liner

`return !!keys.find((key) => disqualifyingKeys.includes(key));`

Reply:
updated function to be more generic.